stdpopsim/ext/selection.py

```diff
@@ -40,6 +39,31 @@ def __attrs_post_init__(self):
                     "Gamma-distributed mutation types (distribution_type='g') "
```

Comment:
I think instead of saying x-distributed mutation types we should change to x-distributed selection coefficients for this MutationType.

Reply:
Agreed!